client/my-sites/site-settings/form-writing.jsx

```diff
@@ -126,7 +126,7 @@ class SiteSettingsFormWriting extends Component {
 				}
 
 				{
-					this.renderSectionHeader( translate( 'Custom content types' ) )
+					this.renderSectionHeader( translate( 'Content types' ) )
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Content Types' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/move-posts-per-page-to-cpt)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).